websites/C/CodAre/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "BBQ",
+    "id": "298888568279924746"
+  },
+  "url": "codare.fun",
+  "description": {
+    "tr": "CodAre'ın kod paylaşım sitesi.",
+    "en": "CodAre's code sharing website."
+  },
+  "service": "CodAre",
+  "version": "1.0.0",
+  "logo": "https://i.hizliresim.com/Tgnky5.png",
+  "thumbnail": "https://i.hizliresim.com/3lOopL.png",
```

Comment:
* Note for verifiers: this is an image uploading website similar to imgur. It's allowed.